sos/sosreport.py

```diff
@@ -126,6 +127,11 @@ def _get_parser():
                         dest="all_logs", default=False,
                         help="collect all available logs regardless "
                              "of size")
+    parser.add_argument("--since", action="store",
+                        dest="since", default=None,
+                        type=lambda d: datetime.strptime(d, '%Y%m%d%H%M%S'),
+                        help="Use in conjunction with --all-logs, only"
+                             " grab files since date. Format: YYYYMMDDHHMMSS")
```

Comment:
I'm not a fan of specifying a timestamp down to the second. I think this should be day-sensitive at most, and I would personally prefer only requiring the user to specifying a number of days ago to collect.

Reply:
This needs to be much more flexible. It's not reasonable to expect users to always type seconds granularity when it's trivial string munging to expand a shorter date string.